views/project.html

```diff
@@ -44,9 +47,18 @@ <h2>{{doc.title}}</h2>
     </select>
 </div>
 
-{% if doc.data %}
 <div class="project clearfix">
-    <h3 id="data">Data</h3>
+    {% if doc.about %}
+    <h3 class="title" id="about">About</h3>
+    <div>
+        {% if doc.about is not none %}
```

Comment:
the first `if doc.about` should be sufficient FYI

Reply:
Doesn't look like this is true, removing the `{% if doc.about is not none %}` line, as well as the corresponding div and `{% endif %}`s broke it

Reply:
It works fine for me.  If `doc.about` is None, it would never even go into this code block.

Reply:
You are saying that I should remove:

`        {% if doc.about is not none %}
            <p>{{doc.about|safe}}</p>
        {% endif %}
`?


Because if I do that, the 'about' section doesn't appear.

Reply:
just the redundant if statement.  you definitely need to render `doc.about`.

I'm saying that instead of this:

```python
    {% if doc.about %}
    <h3 class="title" id="about">About</h3>
    <div>
        {% if doc.about is not none %}
            <p>{{doc.about|safe}}</p>
    {% endif %}
    </div>
    {% endif %}
```

Do this:

```python
    {% if doc.about %}
        <h3 class="title" id="about">About</h3>
        <p>{{doc.about|safe}}</p>
    {% endif %}
```

that above also removes the redundant `div` which I also asked about